content/pages/001_setup/repository_creation.md

```diff
@@ -0,0 +1,41 @@
+Title: Github Repository Creation
+Lang: en
+Category: Setup
+Slug: github-repository-creation
+Summary: Suggestions on how to set up a new Github repository.
+Status: published
+
+At Sage Bionetworks, git is RECOMMENDED for version control and Github RECOMMENDED for storing and sharing code.
+
+## Naming Conventions
+There’s a wide variety of styles in naming repositories, but our RECOMMENDED convention is [kebab-case](https://wiki.c2.com/?KebabCase).
+
+## GitHub Repository Templates
+*Status: ok – maybe some links to examples?*
+
+* Language-agnostic template
+    * Readme
+    * Gitignore (hidden OS files)
+    * Code of Conduct
+    * Contribution Guidelines
+    * Dockerfile (with base alpine/ubuntu?)
+* Language-specific templates
+    * Gitignore (language-specific rules from [gitignore.io](https://www.toptal.com/developers/gitignore))
+    * Package management files (e.g., Pipfile, renv)
+    * Language-specific Dockerfile (e.g., python/rocker base image)
+
+## Repository Structure
```

Comment:
Some common use cases would be useful and encourage more consistent structures (as opposed to tossing a bunch of scripts into directories -- as I've been guilty of! Two obvious ones:

1. Python packages
2. R packages

And we can include links to tools that will auto-create necessary resources and simplify package management, like [devtools](https://www.r-project.org/nosvn/pandoc/devtools.html) for R.